src/Http/api.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+$router->post('api/v1/kpis', [
+    'uses' => 'RequestController@store'
+])->middleware('auth.kpis');
```

Comment:
This does not work (for me) because the middleware alias is never defined. Instead you can directly specify the middleware class:

```suggestion
use Biigle\Modules\Kpis\Http\Middleware\EnsureTokenIsValid;

$router->post('api/v1/kpis', [
    'uses' => 'RequestController@store'
])->middleware(EnsureTokenIsValid::class);
```